testsuite/tests/langkit_support/diagnostics_output/main.adb

```diff
@@ -0,0 +1,70 @@
+with Ada.Exceptions; use Ada.Exceptions;
```

Comment:
Can you add a leading comment to explain what this tests exactly? (describing the condition of the bug for this regression test)